app/js/components/LabTrendsPage.jsx

```diff
@@ -0,0 +1,120 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  PatientHeader,
+  SortableTable,
+} from '@openmrs/react-components';
+
+import RangeCell from "./RangeCell";
+import { fetchLabOrdersTrends } from '../actions/labOrdersAction';
+import { getTestResultDate, getSampleDate } from '../utils/helpers';
+
+import "../../css/lab-orders-trends-page.scss";
+
+export const Cell = ({ columnName, conceptUuid, value }) => {
+  switch (columnName) {
+    case 'SAMPLE DATE': {
+      return (
+        <div className="table_cell sample-date">
+          <span>{getSampleDate(value)}</span>
+        </div>
+      );
+    }
+    case 'RESULT DATE': {
+      return (
+        <div className="table_cell result-date">
+          <span>{getTestResultDate(value)}</span>
+        </div>
+      );
+    }
+    case 'RESULT':
+      return (
+        <div className="table_cell result">
+          <span>{value.value}</span>
+        </div>
+      );
+
+    case 'NORMAL RANGE':
+      return (
+        <RangeCell conceptUUID={conceptUuid} />
+      );
+    default:
+      return null;
+  }
+};
+export class LabTrendsPage extends PureComponent {
+  componentDidMount() {
+    const { fetchLabOrdersTrends, patientHeaderDetail, history: { location: { state } } } = this.props;
+    const conceptUuid = state.uuid;
+    const patientUuid = patientHeaderDetail.uuid;
+    fetchLabOrdersTrends(patientUuid, conceptUuid);
+  }
+
+  render() {
+    const {
+      labOrdersTrend: { result },
+      patientHeaderDetail,
+      history: { location: { state } },
+    } = this.props;
+
+    const fields = ["SAMPLE DATE", "RESULT DATE", "RESULT", "NORMAL RANGE"];
+    const columnMetadata = fields.map(columnName => ({
+      Header:
+  <span className="labs-order-table-head">
+    {columnName}
+  </span>,
+      accessor: "",
+      Cell: data => <Cell {...data} columnName={columnName} conceptUuid={state.uuid} />,
+      className: `lab-order-list-cell-${columnName.replace(' ', '-').toLocaleLowerCase()}`,
+      headerClassName: `lab-order-list-header-${columnName.replace(' ', '-').toLocaleLowerCase()}`,
```

Comment:
I think maybe this shouldn't use the `lab-order-list` class, since its a different table, we can decide to alter styles on the other one which could affect this as well.